File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3-Feb_Solutions_Three/C++/prog3.cpp
#include<iostream>

int main() {
	float dist;
	std::cout << "Enter dist(KM) : ";
	std::cin >> dist;
	if(dist >= 0) {
		std::cout << "dist(m) : " << dist * 1000;
	} else {
		std::cout << "Enter valid det.";
	}
	return 0;
}

File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3-Feb_Solutions_Three/Java/prog3.java
import java.util.Scanner;

class Demo {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.println("Enter dist(KM): ");
		float dist = sc.nextFloat();
		if(dist >= 0) {
			System.out.println("dist(m) : " + dist * 1000);
		} else {
			System.out.println("Not prime.");
		}

	}
}
